GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -594,5 +661,55 @@ private void _RevisionHeader_ContentsResized(object sender, ContentsResizedEvent
             _headerResize = e.NewRectangle;
         }
 
+        private void ApplyLayout()
+        {
+            if (!EnvUtils.IsMonoRuntime())
+            {
+                return;
+            }
+
+            tlpnlRevisionHeader.ColumnStyles[1].SizeType = SizeType.Absolute;
+            tlpnlRevisionHeader.ColumnStyles[1].Width = 38;
+            tlpnlRevisionHeader.ColumnStyles[2].SizeType = SizeType.Absolute;
+            tlpnlRevisionHeader.ColumnStyles[2].Width = 38;
+        }
+        
+        private void commitSignPicture_Click(object sender, EventArgs e)
+        {
+            if (_gpgController == null)
+            {
+                return;
+            }
+            using (var form = new FormSignatureInfo())
+            {
+                form.Text = trsGpgCommitInfo.Text;
+                form.ShowGpgMessage(_gpgController.GetCommitVerificationMessage());
+                form.ShowDialog();
+            }
+        }
+
+        private void tagSignPicture_Click(object sender, EventArgs e)
+        {
+            if (_gpgController == null)
+            {
+                return;
+            }
+            using (var form = new FormSignatureInfo())
+            {
+                form.Text = trsGpgTagInfo.Text;
+                form.ShowGpgMessage(_gpgController.GetTagVerifyMessage());
+                form.ShowDialog();
+            }
+        }
+
+        private void tagSignPicture_OnMouseEnter(object sender, EventArgs e)
+        {
+            ((System.Windows.Forms.PictureBox) sender).Cursor = System.Windows.Forms.Cursors.Hand;
+        }
+
+        private void tagSignPicture_OnMouseLeave(object sender, EventArgs e)
+        {
+            ((System.Windows.Forms.PictureBox)sender).Cursor = System.Windows.Forms.Cursors.Default;
```

Comment:
No need for unsafe casts.
`tagSignPicture.Cursor = System.Windows.Forms.Cursors.Default;`

Reply:
where's the same for `commitSignPicture`?

Reply:
Both use the same method, so the cast to `PictureBox` to access at sender's Cursor and modifying it.
Maybe the name should not start with `tagSignPicture` to be used with all the two PictureBox.

Reply:
in this case please rename the handler, maybe something like `gpgSignatureIcon_OmMouseLeave`.

and cast it in safer manner:

```
var pictureBox = sender as pictureBox;
if (pictureBox != null) 
{
    pictureBox.Cursor = Cursor.Hand;
}
```

Reply:
changed on last commit.